07/28/2024 10:05:13 AM Logging starts
Traceback (most recent call last):
  File "./popular_projects_snippets_dataset/flask/bodies/body_304_refine_9_1.py", line 24, in <module>
    super().__delete__(obj)
RuntimeError: super(): no arguments
Command exited with non-zero status 1
1.53user 4.84system 0:00.64elapsed 985%CPU (0avgtext+0avgdata 107776maxresident)k
0inputs+24outputs (0major+24979minor)pagefaults 0swaps
